src/colorPrinter.py

```diff
@@ -57,3 +57,15 @@ def addstr(self, y, x, text, attr=None):
             attr = self.currentAttributes
 
         self.screen.addstr(y, x, text, attr)
+
+    def clearSquare(self, topY, bottomY, leftX, rightX):
```

Comment:
technically this could be a rectangle since we dont enforce that `rightX - leftX == topY - bottomY` :P